aimmo-reverse-proxy/nginx.conf

```diff
@@ -0,0 +1,13 @@
+
+http {
+    resolver 10.0.0.10;
+    server {
+        listen 80;
+        location ~^/game/([a-zA-Z]+)/? {
+            proxy_pass "http://game-$1.default.svc.cluster.local/";
```

Comment:
I tried this but cannot be used as nginx does not use resolv.conf for dynamic DNS lookups. (It won't use `gethostbyname` because it's blocking.)

The alternative is to try and do something with manually parsing resolv.conf in Dockerfile.